frontend/src/Game.js

```diff
@@ -140,20 +140,25 @@ export default class Game {
             }
 
             // check all enemies, attack first (farthest along path)
+            let towerAttacked = false;
             for (let enemy of this.enemies) {
                 if (!enemy.hp) continue;
                 const path = tower.getProjectilePath(enemy.position, this.gameMap);
-                // console.log(tower, enemy, path)
+
                 if (path.length) {
-                    const projectile = tower.createProjectile(path);
-                    this.addProjectile(projectile);
+                    if (!towerAttacked) { // prevent duplicate projectiles
```

Comment:
Hmm, can you clarify this comment a bit more? Intuitively, if I had a tower that could attack multiple enemies, I imagine I would want a projectile for each enemy. What's the issue this is fixing?

Reply:
We don't have any towers that shoot two "true" projectiles atm, this is to prevent spawning two saws or spike arrays on the same tile.